# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/FragmentOrElement.h
# Commit: 20a5c1f3b35e
# Full Hash: 20a5c1f3b35e80849e0b98ffbe3f9488d703717b
# Author: Frederic Wang <fred.wang@free.fr>
# Date: 2023-11-22 21:50:37
# Regressor Bug: 1807253
# File Overlap Count: 1
# Description:
#   Bug 1807253 - unreliable timing for content-visibility:auto, r=emilio
#   
#   Rewrite implementation of content-visibility: auto as defined in
#   https://github.com/w3c/csswg-drafts/issues/8542
#   
# ==============================================================================

diff -r b90083f581e1 -r 20a5c1f3b35e dom/base/FragmentOrElement.h
--- a/dom/base/FragmentOrElement.h	Mon Nov 20 11:33:16 2023 +0000
+++ b/dom/base/FragmentOrElement.h	Wed Nov 22 13:00:36 2023 +0000
@@ -238,12 +238,16 @@
     /**
      * Whether the content of this element is relevant for the purposes
      * of `content-visibility: auto.
+     * Reflects 'relevant to the user' concept, see
+     * https://drafts.csswg.org/css-contain/#relevant-to-the-user.
      */
     Maybe<ContentRelevancy> mContentRelevancy;
 
     /**
      * Whether the content of this element is considered visible for
      * the purposes of `content-visibility: auto.
+     * Reflects 'proximity to the viewport' concept, see
+     * https://drafts.csswg.org/css-contain/#proximity-to-the-viewport.
      */
     Maybe<bool> mVisibleForContentVisibility;
 